### DISCUSSION TITLE
GPL License

### DISCUSSION THREAD

**Discussion by hakontro** [bkkas/elhub-python-sdk#11] [General]
Hi,

Just realized the SDK has the GPL3.0 license, which among other things states:

> Permissions of this strong copyleft license are conditioned on making available complete source code of licensed works and modifications, **which include larger works using a licensed work**, under the same license.

Interested in hearing about the decision, and whether or not it can be changed? We are super happy to contribute to the development and quality of the SDK, but it's a hard requirement for us that we can build things **on top of it** which don't have to be open sourced.

**Comment by jgc31416** [comment]
Hi @hakontro yes, this is a fair point, GPL3.0 is too restrictive, it seems LGPL v3 is a better match for libraries, as it allows to use them without turning the whole project into GPL 3. I will check if we can change it, I don't think that should be an issue.

**Reply by hakontro** [reply]
Great, thanks! LGPL looks fine for our case :)

**Reply by jgc31416** [reply]
Hi @hakontro we have changed the license type, now it is using LGPL v3 :)
